docs/network-policy.md

```diff
@@ -0,0 +1,361 @@
+            ns: demo
+      - podSelector:
+          matchLabels:
+            app: demo
+  ``` 
+  Then there would be two elements in the `from` array which allows connections from Pods labeled `app=demo` **OR** and Pod from namespaces with the label `app=demo`
+
+  Now let's have a look at some of the OVN resources that are created along with this Network Policy 
+
+  For all worker nodes we can see the UUIDs of the logical ports corresponding to the pods `client1`, `client2`, and `server` 
+
+  **ovn-worker**
+
+  ```
+  [root@ovn-control-plane ~]# ovn-nbctl lsp-list ovn-worker
+  edb290cf-b250-4699-b102-7acbb6300dc9 (default_client1)
+  c754a19d-1e8c-4415-99b9-66fdcdaed196 (demo_server)
+  24c789a2-fc4b-42a5-bb16-5b1c19490b50 (k8s-ovn-worker)
+  484b2004-a5c1-447c-b857-eb8e524a73f3 (kube-system_coredns-f9fd979d6-qp6xd)
+  45163af0-08c2-4d42-9fc7-7b0ddc935bd8 (local-path-storage_local-path-provisioner-78776bfc44-lgzdf)
+  0bb378f1-4e89-46a8-a455-7a891f64c7c8 (stor-ovn-worker)
+  ```
+  **ovn-worker2**
+
+  ```
+  [root@ovn-control-plane ~]# ovn-nbctl lsp-list ovn-worker2
+  d5030b96-1163-4ed0-90f8-41b3831d2a0b (default_client2)
+  4da8fb64-0a43-4d76-a7ba-18941c078862 (k8s-ovn-worker2)
+  37f58eeb-8c1a-437b-8b21-bcc1337b2e3f (kube-system_coredns-f9fd979d6-628xd)
+  65019041-51b2-4599-913d-7f01c8eaa394 (stor-ovn-worker2)
+  ```
+
+  **Port Groups** 
+
+  ```
+  [root@ovn-control-plane ~]# ovn-nbctl find port
+  _uuid               : 2b74086c-9986-4f4d-8c97-3388625230e9
+  acls                : []
+  external_ids        : {name=clusterPortGroup}
+  name                : clusterPortGroup
+  ports               : [24c789a2-fc4b-42a5-bb16-5b1c19490b50, 4da8fb64-0a43-4d76-a7ba-18941c078862, e556e329-d624-473a-8827-f022c17a8f60]
+
+  _uuid               : a132ecce-dbca-4989-87f7-96e2f0b62a2c
+  acls                : [b4e57f83-8b8f-4b37-b5e5-1f82704c49c4]
+  external_ids        : {name=demo_allow-from-client}
+  name                : a13757631697825269621
+  ports               : [c754a19d-1e8c-4415-99b9-66fdcdaed196]
+
+  _uuid               : a32d9dda-d7fb-4ae8-b6a9-3af17d62aa7f
+  acls                : [510d797c-6302-4171-8a08-eeaab67063f4, f9079cce-29aa-4d1b-b36b-ca39933ad4e6]
+  external_ids        : {name=ingressDefaultDeny}
+  name                : ingressDefaultDeny
+  ports               : [c754a19d-1e8c-4415-99b9-66fdcdaed196]
+
+  _uuid               : 896a80ff-46f7-4837-a105-7b52cee0c625
+  acls                : [660b10ea-0f2e-49cb-b620-ca4218e87ac6, 9bb634ff-cb69-44b6-a64d-09147cf337b5]
+  external_ids        : {name=egressDefaultDeny}
+  name                : egressDefaultDeny
+  ports               : []
+  ```
+
+  Notice that the port corresponding to the pod `server` is included in the `ingressDefaultDeny` port group 
+
+  To bypass the ingress default deny and allow traffic from pod `client1` in namespace `demo` as specificed in the network policy, an address set is created containing the ip address for the pod `client1`  
+
```

Comment:
Maybe include the command here `ovn-nbctl list address_set`

Reply:
Added a note